rest-api/src/main/java/gov/cms/qpp/conversion/api/config/SecurityConfig.java

```diff
@@ -16,6 +18,9 @@
 
 	private static final String CPC_WILDCARD = "/cpc/**";
 
+	@Value("${ORG_NAME:" + JwtAuthorizationFilter.DEFAULT_ORG_NAME + "}")
```

Comment:
why not `@Value("${ORG_NAME}")`?

Reply:
@clydet you can give a default value with a :